package.json

```diff
@@ -73,6 +83,17 @@
     "url": "git+https://github.com/mozilla/blurts-server.git"
   },
   "scripts": {
+    "js:minify": "minify public/dist/app.min.js -o public/dist/app.min.js",
+    "js:concat": "concat -o public/dist/app.min.js public/js/*.js",
+    "css:minify": "cleancss -o public/dist/app.min.css public/dist/app.min.css",
+    "css:autoprefixer": "postcss -u autoprefixer --autoprefixer.browsers '> 5%, ie 11' -r public/dist/app.min.css --no-map",
+    "css:vars": "postcss -u postcss-css-variables -o public/dist/app.min.css public/css/app.css",
+    "build:js": "run-s js:concat js:minify",
+    "build:css": "run-s css:vars css:autoprefixer css:minify",
+    "watch:js": "onchange 'public/js/monitor.js' -- npm run build:js",
+    "watch:css": "onchange 'public/css/app.css' -- npm run build:css",
+    "watch:all": "run-p watch:css watch:js",
+    "build:all": "run-s build:js build:css watch:all",
```

Comment:
Drop `watch:all` from this - when I ran this I expected the `build:all` command to return.